src/js/services/optimizely.js

```diff
@@ -181,14 +182,13 @@ const trackExperimentViewed = (
     // store experiment participation in localstorage
     storeExperimentParticipation(orgId, experimentKey, variationName);
   }
-};
+}
 
 // isExperimentAlreadyViewed checks in the localstorage if we already
 // marked the experiment as viewed
-const isExperimentAlreadyViewed = (orgId, experimentKey) => {
+export function isExperimentAlreadyViewed(orgId, experimentKey) {
```

Comment:
why remove the `const` and the `=>` function? 

This should work:
`export const isExperimentAlreadyViewed = (orgId, experimentKey) => {`

Reply:
Both work the same it was more keeping it consistent. It seems like some files are written one way and some are written another. I was matching how the most recent tests were done with highlightURLHash, Users and sidebar. I will add it to blue tape item but put it how you suggested.